Doc/library/pyexpat.rst

```diff
@@ -614,6 +614,16 @@ otherwise stated.
 
 .. method:: xmlparser.ExternalEntityRefHandler(context, base, systemId, publicId)
 
+   .. warning::
+
+      Implementing a handler that accesses local files and/or the network
+      may create a vulnerabilitiy to
+      `external entity attacks <https://en.wikipedia.org/wiki/XML_external_entity_attack>`_
+      if :class:`xmlparser` is used with user-provided XML content.
+      Please reflect on your `threat model <https://en.wikipedia.org/wiki/Threat_model>`_
+      before implementing this handler.
+      By default, :class:`xmlparser` is safe from these threats.
```

Comment:
Thank you, this is better.

But I think the last sentence is redundant. We don't have them in other notes, this is default.